design/sketch.webidl

```diff
@@ -89,12 +89,66 @@ enum GPUTextureDimension {
     "3d"
 };
 
+// Texture formats
+// The name of the format specifies the order of components, bits per component, and data type for
+// the component.
+//     r, g, b, a = red, green, blue, alpha
+//     unorm = unsigned normalized
+//     snorm = signed normalized
+//     uint = unsigned int
+//     sint = signed int
+//     float = floating point
+// If the format has the "-srgb" suffix, then sRGB gamma compression and decompression are
+// applied during the reading and writing of color values in the pixel.
+
 enum GPUTextureFormat {
-    "r8g8b8a8-unorm",
-    "r8g8b8a8-uint",
-    "b8g8r8a8-unorm",
-    "d32-float-s8-uint"
-    // TODO other formats
+    /* Normal 8 bit formats */
+    "a8unorm",
```

Comment:
note: this one isn't supported by Vulkan

Reply:
In theory it is if you use component swizzling behind the user's back

Reply:
Yes, for this particular case we could emulate the format. This is only possible since A8 behaves like R8 for all scenarios where an image view is not involved.

Reply:
I think we should remove it. Sorry I didn't notice this before.

We can always add it back later if we think it's worth it.